src/lib/components/documents/ResultsList.svelte

```diff
@@ -145,12 +149,17 @@
         }}
       >
         {#if loading}
-          Loading &hellip;
+          {$_("common.loading")}
         {:else}
-          Load more
+          {$_("documents.more")}
         {/if}
       </Button>
     {/if}
+
+    {#if error}
+      <p class="error">{error}</p>
+      <p class="error">{$_("documents.retry")}</p>
```

Comment:
"Retry" should give the user some kind of actionable step. Since we have the load function available, this could be a button to call it again for the URL. Instead of "Please try again," the UI string could just be "Try Again"